<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>NanumSquare</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>정보입력</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>24</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>이       름 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtName">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhLatinOnly</set>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtPhone">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>49</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly</set>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>53</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>전화번호 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtEmail">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>79</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhEmailCharactersOnly</set>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>83</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>이  메  일 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>113</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>주       소 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtAddress">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>109</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly|Qt::ImhLatinOnly</set>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnNew">
        <property name="geometry">
         <rect>
          <x>445</x>
          <y>107</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>신규</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSave">
        <property name="geometry">
         <rect>
          <x>527</x>
          <y>107</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>저장</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>621</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="tblAddress"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="stbCurrent"/>
 </widget>
 <resources/>
 <connections/>
</ui>
